standard/stream_links.lua

```diff
@@ -87,13 +103,25 @@ function StreamLinks.processStreams(forwardedInputArgs)
 		)
 
 		if String.isNotEmpty(streamValue) then
+			local platform = platformName
+			local languageCode = 'en'
+			local count = 1
 			-- stream has no platform
-			if platformName ~= 'stream' then
-				local lookUpPlatform = StreamLinks.streamPlatformLookupNames[platformName] or platformName
+			if platform ~= 'stream' then
+				if StreamLinks.streamPlatformLookupNames[platform] then
+					count = tonumber(platform:match('(%d+)$'))
```

Comment:
Explanation of the regex in a comment would be useful

Reply:
The regex has been removed in the rework, as an improved is being used.